openwisp_firmware_upgrader/signals.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+from django.db.models.signals import post_save
+from django.dispatch import Signal, receiver
+
+from .websockets import DeviceUpgradeProgressPublisher, UpgradeProgressPublisher
+
+logger = logging.getLogger(__name__)
+
+upgrade_log_line = Signal()
```

Comment:
I don't love this name.

What about `firmware_upgrader_log_updated`?
This name is pretty self explanatory.

Please update the docs to mention this signal. See:

https://openwisp.io/docs/dev/controller/developer/utils.html#signals
https://openwisp.io/docs/dev/monitoring/developer/utils.html#signals